tox.ini

```diff
@@ -35,20 +35,19 @@ norecursedirs = .* docs requirements
 
 [testenv]
 deps =
-    django18: Django>=1.8,<1.9
     django111: Django>=1.11,<2.0
     django20: Django>=2.0,<2.1
+    django21: Django>=2.1,<2.2
+    django22: Django>=2.2,<2.3
```

Comment:
we need to use `django.txt` file for Django2.2 version as we did in other repos. Update `MakeFile` too for this change. For reference see https://github.com/edx/portal-designer/pull/84

Reply:
@jmbowman against which version of Django do we want to deploy this package? `1.11` or `2.2`?